.github/workflows/Tests.yaml

```diff
@@ -170,8 +170,14 @@ jobs:
           - build_type: Debug
             EXTRA_CXX_FLAGS: "-DBLAZE_USE_VECTORIZATION=0"
           - compiler: clang-8
-            build_type: Release
+            build_type: Debug
+            # When building with ASAN we also need python bindings to be
+            # disabled because otherwise we get link errors:
+            # ImportError: bin/python/spectre/Informer/_PyInformer.so:
+            # undefined symbol: __asan_option_detect_stack_use_after_return
```

Comment:
Should we investigate this at some point? If so, please open an issue and link it here.

Reply:
Yes, #1890

Reply:
thanks, could you replace lines 176-177 with a link to the issue? (it was probably not clear from my comment that with "here" I meant those lines in the code 😄)

Reply:
Okay, pushed a fixup that does this.